red_queen/games/applications/run_hamiltonian_sim.py

```diff
@@ -0,0 +1,209 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the MIT License.
```

Comment:
Looks good! I'm pretty sure that we're only under the Apache license, but check with Matthew to make sure, otherwise, it works perfectly for me :)
```suggestion
# Part of Red Queen Project.  This file is distributed under the Apache 2.0 License.
```